gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -4,21 +4,27 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>GPII Canopy MatchMaker tests</title>
 
-        <link rel="stylesheet" media="screen" href="../../../../../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" media="screen" href="/node_modules/infusion/tests/lib/qunit/css/qunit.css" />
```

Comment:
Thanks for adding the alternative with relative paths. This doesn't work with my setup yet because the inclusion of `/testem.js` and `/coverage/client/coverageSender.js` at line 26, 27 is still using absolute paths.